docs/root/intro/arch_overview/ssl.rst

```diff
@@ -85,6 +85,29 @@ standard Debian installations. Common paths for system CA bundles on Linux and B
 See the reference for :ref:`UpstreamTlsContexts <envoy_api_msg_auth.UpstreamTlsContext>` and
 :ref:`DownstreamTlsContexts <envoy_api_msg_auth.DownstreamTlsContext>` for other TLS options.
 
+.. _arch_overview_ssl_multi_cert: 
+
+Multiple certificates
+---------------------
+
+:ref:`UpstreamTlsContexts <envoy_api_msg_auth.UpstreamTlsContext>` support multiple TLS
```

Comment:
I think it's the reverse? We support multiple contexts for downstream but not for upstream?